src/main/webapp/WEB-INF/themes/desktop/sass/sections/_header.scss

```diff
@@ -379,6 +379,8 @@ $width-nav: $block-grid-elements - $width-logo;
       a {
         padding-top: 0;
         padding-bottom: 0;
+        vertical-align: middle;
+        line-height: 28px;
```

Comment:
should we make this rem-calc(28)? to be consistent

Reply:
Also:    align-items: center;
    display: inline-flex;

will make it possible to be more flexible - no  pun intended and not have to set the height